docs/source/add_python_operator.rst

```diff
@@ -15,23 +17,25 @@ Adding a simple Python operator
 
 A simple Python operator is one that adds only a single underlying SMAUG
 operator. For the purposes of this tutorial, we'll continue using the
-`MyCustomOperator` operator we built in the C++ tutorial, which implements an
-elementwise add. This operator is as simple as it can get: it takes just two
-input tensors and no additional parameters and produces a single output tensor.
-
-First, go to `smaug/python/ops <doxygen_html/node_8proto_source.html>`. All
-operators are defined in Python files here, organized by operator type. For
-example, `math_ops` contains arithmetic operators like add/multiply/etc, while
-`array_ops` contains operators for manipulating tensors by
-reshaping/concatentating/etc.  In practice, we would add a new operator to an
-existing file there. But for the purposes of this tutorial, we'll create a
-brand new file called `my_custom_operator.py`.
+:code:`MyCustomOperator` operator we built in the C++ tutorial, which
+implements an elementwise add. This operator is as simple as it can get: it
+takes just two input tensors and no additional parameters and produces a single
+output tensor.
+
+First, go to :code:`smaug/python/ops`. All operators are defined in Python
```

Comment:
Oh, looks like you found a better way to achieve that :)

Reply:
This actually just achieves fixed-width formatting, not a link.